pages/techgenius/tg-hardware-setup.md

```diff
@@ -15,11 +15,9 @@ The Planet system has two hardware configurations: Raspberry Pi and router confi
 
 **Background** What is a Raspberry Pi? A Raspberry Pi (also referred to as an RPi) is low-cost and credit card-sized computer that stores data via microSD card. RPis are often used to teach people about computer basics and computer programming. There are also many higher-level applications that utilize the RPi due to low costs and considerable capabilities. In our case, it’s a very easy, small, cheap, and portable tool for our library. Check out [this video](https://www.youtube.com/watch?v=_cviTxswW8c) to learn more about the RPi.
 
-Below are two images of the RPi with and without a case. On the right is an image of an RPi in a case.
+Below are two images of the RPi with and without a case. On the right is an image of an RPi in a case and on the left is an image of RPI without a case.
```

Comment:
Seems like the order is reversed here. But is it really necessary to explain these are the images with and without the case?

Reply:
I'll get right to fixing the order. And you are right, it isn't necessary to explain that the image is on the left or right. I'l have that removed and make a new commit